src/components/Modal/Modal.vue

```diff
@@ -281,6 +281,13 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		/**
+		 * Makes the modal flex
+		 */
+		flex: {
```

Comment:
Oh please no. That's even worse :see_no_evil:

Reply:
Ok, we just clarified and using v-deep is ok for this use case. At first I thought v-deep was forbidden.
I'll adjust.